tests/testthat/test-tar-stars.R

```diff
@@ -0,0 +1,21 @@
+# test_that() #Included to make RStudio recognize this file as a test
+
+skip_if_not_installed("sf")
```

Comment:
I think this is unecessary, again because `sf` is a hard dependency of `stars`

Reply:
done in https://github.com/njtierney/geotargets/pull/33/commits/0e20c172f574efed855625ec06219c40eade9917